Updated docs for active directory (#9726)
DataDog/integrations-core: active_directory/README.md

#### Log collection

> This integration uses [win32 event logs integration][10] as the transport mechanism for active directory logs.

Reviewer: ```suggestion
This integration uses the [Win 32 event logs integration][10] as the transport mechanism for Active Directory logs.
```

> 1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:

Reviewer: ```suggestion
1. Collecting logs is disabled by default in the Datadog Agent. Enable it in `datadog.yaml`:
```

   ```yaml
   logs:
     - type: file
       path: /path/to/my/directory/file.log
       source: ruby

>        service: "<MY_SERVICE>"

Reviewer 2: Should we extend the logs with the new types instead of removing the other example?

Author: We should not be relying on Ruby since it may send us logs in a different format breaking our pipeline and dashboards. Our native support for event viewer logs is much better suited for that purpose.

   ```

>    Change the `channel_path` parameter value and configure them for the type of AD logs (e.g. `security`, `application`, `system` etc)

Reviewer: ```suggestion
   Specify the the type of AD logs (for example, `security`, `application`, or `system`), and provide a value for the `channel_path` parameter.
```

>    Add multiple blocks of the above configuration with different channels if you wish to forward all of them.

Reviewer: ```suggestion
   Add multiple blocks of the above configuration with different channels if you wish to forward more.
```

>    The logs can be filtered using [win32 transport filters][11].  

Reviewer: ```suggestion
   Filter logs using [Win 32 transport filters][11].  
```
